lib/overcommit/hook/pre_commit/yaml_lint.rb

```diff
@@ -11,7 +11,10 @@ def run
       if result.success?
         :pass
       else
-        return [:warn, result.stdout]
+        if result.stdout.include? "error"
+          return [:fail, result.stdout]
+        else
+          return [:warn, result.stdout]
```

Comment:
You are missing an `end` statement after this. This is a syntax error, which is why the specs fail.

Reply:
Outch this doesn't look good indeed. I corrected.